docs/guides/backends/local/index.md

```diff
@@ -14,264 +14,141 @@ section_menu_id: guides
-
-Below, the YAML of a sample `Repository` crd that uses an [azureDisk](https://kubernetes.io/docs/concepts/storage/volumes/#azuredisk) as a backend.
-
-```yaml
-apiVersion: stash.appscode.com/v1alpha1
-kind: Repository
-metadata:
-  name: local-repo-with-azuredisk
-  namespace: demo
-spec:
-  backend:
-    local:
-      mountPath: /safe/data
-      azureDisk:
-        diskName: stash.vhd
-        diskURI: https://someaccount.blob.microsoft.net/vhds/stash.vhd
-    storageSecretName: local-secret
-```
-
-Create the `Repository` we have shown above using the following command,
-
-```bash
-$ kubectl apply -f https://github.com/kubestash/docs/raw/{{< param "info.version" >}}/docs/guides/backends/local/examples/azureDisk.yaml
-repository/local-repo-with-azuredisk created
-```
-
-##### StorageOS as Backend
-
-Below, the YAML of a sample `Repository` crd that uses a [storageOS](https://kubernetes.io/docs/concepts/storage/volumes/#storageos) volume as a backend.
-
-```yaml
-apiVersion: stash.appscode.com/v1alpha1
-kind: Repository
-metadata:
-  name: local-repo-with-storageos
-  namespace: demo
-spec:
-  backend:
-    local:
-      mountPath: /safe/data
-      storageos:
-        volumeName: stash-vol01 # The `stash-vol01` volume must already exist within StorageOS in the `demo` namespace.
-        fsType: ext4
-    storageSecretName: local-secret
-```
-
-Create the `Repository` we have shown above using the following command,
-
-```bash
-$ kubectl apply -f https://github.com/kubestash/docs/raw/{{< param "info.version" >}}/docs/guides/backends/local/examples/storageOS.yaml
-repository/local-repo-with-storageos created
-```
-
-##### EmptyDir volume as Backend
-
-Below, the YAML of a sample `Repository` crd that uses an [emptyDir](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir) as a backend.
-
-```yaml
-apiVersion: stash.appscode.com/v1alpha1
-kind: Repository
-metadata:
-  name: local-repo-with-emptydir
-  namespace: demo
-spec:
-  backend:
-    local:
-      mountPath: /safe/data
-      emptyDir: {}
-    storageSecretName: local-secret
-```
-
-Create the `Repository` we have shown above using the following command,
-
-```bash
-$ kubectl apply -f https://github.com/kubestash/docs/raw/{{< param "info.version" >}}/docs/guides/backends/local/examples/emptyDir.yaml
-repository/local-repo-with-emptydir created
+backupstorage.storage.kubestash.com/local-storage-with-nfs created
 ```
 
->**Warning:** Data of an `emptyDir` volume is not persistent. If you delete the pod that runs the respective stash container, you will lose all the backed up data. You should use this kind of volumes only to test backup process.
+>For NFS backend, KubeStash may have to run the network volume accessor deployments in privileged mode to provide KubeStash CLI facilities. In this case, please configure network volume accessors by following the instruction [here](/docs/setup/install/troubleshooting/index.md#configuring-network-volume-accessor).
```

Comment:
For network volumes such as NFS, KubeStash needs to deploy a helper network volume accessor deployment in the same namespace as the BackupStorage. This deployment mounts the NFS volume, allowing the CLI to interact with the backend. You can configure the network volume accessor by following the instructions [here](/docs/setup/install/troubleshooting/index.md#configuring-network-volume-accessor).